README.md

```diff
@@ -1,13 +1,27 @@
-# Grafana LLM App (Experimental)
+# Grafana LLM App (Public Preview)
 
-A Grafana plugin designed to centralize access to LLMs, providing authentication, rate limiting, and more.
+A Grafana plugin designed to centralize access to LLMs, providing authentication, proxying, streaming, and custom extensions.
 Installing this plugin will enable various pieces of LLM-based functionality throughout Grafana.
 
-Note: This plugin is **experimental**, and may change significantly between
-versions, or be deprecated completely in favor of a different approach based on
-user feedback.
+Note: The Grafana LLM App plugin is currently in [Public preview](https://grafana.com/docs/release-life-cycle/). Grafana Labs offers support on a best-effort basis, and there might be breaking changes before the feature is generally available.
 
-## Installing this plugin
+## Install the plugin on Grafana Cloud
+
+Prerequisites:
+- Any Grafana Cloud environment (including Free)
+- API connection details from an account with [OpenAI](https://platform.openai.com) or [Azure OpenAI](https://oai.azure.com/)
+
+Steps:
+1. In your Grafana instance, open Administration → Plugins
```

Comment:
Would pointing to the actual link here simplify this step? (e.g. sign in -> go to https://grafana.com/grafana/plugins/grafana-llm-app/ -> install in selected instance)

Reply:
Yeah I think we should definitely link to that URL, saves the user a few clicks.

Reply:
It is a little faster, but they're going to need to find the LLM plugin inside their instance anyway to confirm the install worked, and then to configure it. :shrug:

I am wondering if it's not great for us to walk through plugin installation steps when they are elsewhere in docs (and might change in the future and break our steps here): https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-a-plugin

Reply:
I think I found a compromise -- I included the direct plugin link in the steps, so someone who knows how this all works can skip ahead. Thoughts?

Reply:
Looks good to me, I think I'd be able to follow these instructions 😅